python/docs/_scripts/README.md

```diff
@@ -0,0 +1,36 @@
+## _scripts
+
+The files in this folder are used by quartodoc to
+generate custom output for the Python API docs.
+
+#### custom renderer (and summarizer)
+
+Start here if you want to change any of the markdown generated for our API docs.
+
+`renderer.py` and `summarizer.py` are based off of the [default quartodoc renderer](https://github.com/machow/quartodoc/blob/main/quartodoc/renderers/md_renderer.py),
+split into 2 files.
+
+Both of these files use the `@dispatch` decorator to overload the same method
+name on many different input types. It can be tricky to find the correct
+method to update.
+
+#### custom builder
+
+Start here if you want to change how & where we output individual markdown files
+for our API docs.
+
+* `builder.py` iterates over a passed config file and then uses `renderer.py` and `summarizer.py`
+to output markdown for the API docs. It based off the [default quartodoc builder](https://github.com/machow/quartodoc/blob/v0.6.3/quartodoc/autosummary.py#L367),
+but is heavily modified.
+* `gen_reference.py` sets up logging, and calls `builder.py` with the `../_reference.yml` config file
+
+Note that we output files in a VERY different way than the default quartodoc executable behaves.
+For instance, we rely on the path of each file to build the side-bar navigation instead of
```

Comment:
Is this something we should try to change, or are we generally happy with how we have this / believe it won't be strangely broken by future quarto changes?

Reply:
this is quartodoc and not quarto. I'm pretty sure this won't break on future updates to quartodoc.

basically I'm making quartodoc generate output that is closer to how quarto wants things than quartodoc itself does.